wagtail_extensions/templates/wagtail_extensions/blocks/images.html

```diff
@@ -0,0 +1,8 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<div class="row">
+{% for image in value.images %}
+    <div class="col-md-{{ column_width }}">
+        {% image image original class="img-responsive img-thumbnail" %}
```

Comment:
Rendering the original image is risky. You can end up with huge files being sent down, and you will also have display issues if the images chosen have different aspect ratios.

I think we need to make some assumptions - i.e., thumbnail and use a fixed aspect ratio. Projects can customise this in the template.